pyproject.toml

```diff
@@ -46,6 +46,20 @@ shellcheck-py = "^0.7.1"
 yamlfmt = {url = "https://github.com/jumanjihouse/pre-commit-hook-yamlfmt/archive/0.1.0.tar.gz"}
 yamllint = "*"
 
+[tool.poetry.extras]
+cdk = [
+    "aws-cdk.aws-s3",
+    "aws-cdk.aws-iam",
+    "aws-cdk.aws-dynamodb",
+    "aws-cdk.aws-lambda",
+    "awscli",
+    "cattrs",
+]
+datasets_endpoint = [
```

Comment:
Any reason why not to use hyphen datasets-endpont? Hyphen is already used in this file (see: tool.poetry.dev-dependencies)

Reply:
Hmm, good point. Done.